### PR TITLE
Add cleanslate@0.10.1 w/ git auto-update

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #10263
Related issue(s): # #

@cdnjs/intern3 Please help me reviewing this PR. Thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/premasagar/cleanslate
 * Official website (optional, not the repository): http://cleanslatecss.com
 * License and its reference: MIT, [ref](https://github.com/premasagar/cleanslate/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 17
   - Count of stars: 401
   - Count of forks: 49

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by hare1039** [comment]
@pvnr0082t Please help me reviewing this PR. Thanks.

File: ajax/libs/cleanslate/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "cleanslate",
+  "filename": "cleanslate.min.css",
+  "version": "0.10.1",
+  "homepage": "http://cleanslatecss.com",
+  "description": "Extreme CSS reset stylesheet",
+  "keywords": [
+    "reset css"
```
Comment (pvnr0082t) [inline ajax/libs/cleanslate/package.json]: Why not use the keywords from upstream? https://github.com/premasagar/cleanslate/blob/master/component.json#L6

File: ajax/libs/cleanslate/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "license": "MIT",
+  "author": "Premasagar Rose <p@dharmafly.com>",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/premasagar/cleanslate.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "cleanslate.css"
```
Comment (pvnr0082t) [inline ajax/libs/cleanslate/package.json]: @hare1039 We should use `cleanslate*` to be more flexible.

File: ajax/libs/cleanslate/package.json (hunk 1 above)
Comment (hare1039) [inline ajax/libs/cleanslate/package.json]: @pvnr0082t Sorry I didn't read through this file. I amend the package.json, please review this PR again. Thanks.

**Comment by pvnr0082t** [comment]
@hare1039 `amend` does not mean you fixed the problem. I suggest you use some words like `solve` or `fix`, etc.

**Pull request merged**